src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+function addElement(message, text) {
+  body.insertAdjacentHTML('beforeend', `
+
+    <div class="${message}">${text}</div>
+ `);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    setTimeout(() => reject(new Error('error')), 3000);
+  });
```

Comment:
```suggestion
    setTimeout(reject, 3000);
```

just call reject after 3 seconds